<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblVideoSource">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Source</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCameraStart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblError">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>FileStatus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCameraStop">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Camera</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_StartCapturing">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Capturing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
